polyseam/cndi
# [Enhancement:] Make YAML the default syntax

# Related issue
#54 

# Description

- [x] cndi Template files made by the CNDI team ported to YAML
- [x] older versions of CNDI can still find the old JSON templates until we deprecate them (right away?)
- [x] `cndi init -t` reads YAML or JSON and outputs YAML
- [x] `cndi ow` can process YAML or JSON 

# Test Instructions

1. initialize a new project with `cndi-next -i -d -t ec2/airflow` using this branch
2. ensure the resulting cluster repo has a `cndi-config.yaml`
3. initialize a new project in the exact same way using the `main` branch
4. compare the contents of `cndi/` in both new projects are the same
5. make a change to cndi-config.yaml in the first project, then call `cndi-next ow` using this branch 
6. make a change to cndi-config.jsonc in the second project, then call `cndi ow` using the main branch
7. ensure once again that `cndi/` has the same contents
8. rinse and repeat for as many templates as possible, adding a comment here when you have completed one such test for a template

# Notes

- 5673/6025 lines of code added here are purely the new yaml versions of templates, and do not represent real complexity

# Code of Conduct
By submitting this Pull Request, you agree to follow our
[Code of Conduct](https://github.com/polyseam/cndi/blob/main/CODE_OF_CONDUCT.md)
